src/components/Skeleton/Skeleton.module.css

```diff
@@ -4,24 +4,24 @@
 
 @keyframes bg-wipe {
   0% {
-    background-position: 0% 0%;
+    background-position: right 0%;
   }
   100% {
-    background-position: 100% 0%;
+    background-position: left 0%;
   }
 }
 
 /**
  * Skeleton
  */
 .skeleton {
-  border-radius: calc(var(--eds-border-radius-lg) * 1px);
-  background-color: var(--eds-theme-color-background-neutral-medium);
+  border-radius: calc(var(--eds-border-radius-150) * 1px);
```

Comment:
Did EDS change from using symbolic sizes (e.g. lg) to numeric (e.g. 150)? What does 150 mean?